sdcm/nemesis.py

```diff
@@ -227,6 +227,21 @@ def disrupt_nodetool_decommission(self, add_node=True):
                     self.cluster.wait_for_init(node_list=new_nodes)
                     self.reconfigure_monitoring()
 
+    def disrupt_nodetool_enospc(self, sleep_time=30):
+        self._set_current_disruption('Enospc %s' % self.target_node)
+        if not os.path.ismount('/var/lib/scylla'):
+            self.log.error("Scylla doesn't use an individual storage, skip enospc test")
+            return
+        dd_cmd = 'dd if=/dev/zero of=/var/lib/scylla/occupy_rest_space'
+        self.log.debug('Cost rest space on /var/lib/scylla/ by {}'.format(dd_cmd))
+        process.run(dd_cmd, ignore_status=True)
+        assert result.exit_status == 1 and 'No space left on device' in result.stderr
+        self._run_nodetool('nodetool status', self.target_node)
+        time.sleep(sleep_time)
```

Comment:
I think the service is still up, but the node is drained (not serving requests).
In nodetool it should appear as DN (only for other nodes, the target node will see itself as UN).

We should wait for "I/O error" and "Drained".